scripts/cfos/make_dataset_data_tables.py

```diff
@@ -143,57 +108,98 @@ def load_excel_input(excel, expected_dataset_cols):
+    
 
-    # get all cols (attributes) from all tables for dataset
-    # returns nested list of lists with column names
-    nested_dataset_cols = [schema_dict[dataset_name][table] for table in dataset_table_names]
-    # unnest for flat list of columns to parse from excel file
-    expected_dataset_cols = [col for table_cols in nested_dataset_cols for col in table_cols]
+    schema_dict = config_dict["schema_definitions"]
+    column_dict = config_dict["fields"]
 
-    print("Success: Expected list of columns has been determined.")
-    return expected_dataset_cols, dataset_table_names, schema_dict
+    print("Success: Config file parsed.")
+    return schema_dict, column_dict
 
 
 if __name__ == "__main__":
 
     parser = argparse.ArgumentParser(description='Create, validate, and upload (optionally ) Terra dataset specific data tables to a Terra workspace.')
 
-    parser.add_argument('-d', '--dataset_name', required=True, type=str, help='Dataset type to apply validations and data table structure. ex. proteomics, atac-seq, transcriptomics, singe-cell')
-    parser.add_argument('-x', '--excel', required=True, type=str, help='CFoS data excel file (.xlsx) - based on CFoS data intake template file.')
-    parser.add_argument('-j', '--schema', required=True, type=str, help='Dataset schema file (.json) - defines per dataset tables and associated attributes.')
-    parser.add_argument('-p', '--project', required=False, type=str, help='Terra workspace project (namespace).')
-    parser.add_argument('-w', '--workspace', required=False, type=str, help='Terra workspace name.')
-    parser.add_argument('-v', '--validate', required=False, action='store_true', help='Set parameter to run only validation on input excel file.')
-    parser.add_argument('-u', '--upload', required=False, action='store_true', help='Set parameter to upload data table files to workspace. default = no upload')
+    # REK TODO: update so JSON is assumed to live in an internal directory location and not as an input file. 
+    # Goal: allow qualified people to make easy updates to schemas, but don't encourage random users to play with it.
+
+    parser.add_argument(
+        '-d', 
+        '--dataset_name', 
+        required=False,
+        type=str, 
+        help='Dataset type to apply validations and data table structure. ex. proteomics, atac-seq, transcriptomics, singe-cell'
+    )
+    parser.add_argument(
+        '-x', 
+        '--excel', 
+        required=False,
+        type=str, 
+        help='CFoS data excel file (.xlsx) - based on CFoS data intake template file.'
+    )
+    parser.add_argument(
+        '-p', 
+        '--project', 
+        required=False, 
+        type=str, 
+        help='Terra workspace project (namespace).'
+    )
+    parser.add_argument(
+        '-w', 
+        '--workspace', 
+        required=False, 
+        type=str, 
+        help='Terra workspace name.'
+    )
+    parser.add_argument(
+        '-v', 
+        '--validate', 
```

Comment:
minor: I'd call this `validate_only` since its action is exclusionary and you don't need this flag if you want to validate and create TSVs